File: Array/4. CheckIfArrayIsSorted.ts
function checkIfArrayIsSorted(arr: number[]) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] > arr[i + 1]) {
      return false;
    }
  }
  return true;
}

console.log(checkIfArrayIsSorted([5, 4, 6, 7, 8]));
console.log(checkIfArrayIsSorted([1, 2, 3, 4, 5]));

File: Array/4. checkIfArrayIsSorted.py
def isArraySorted(arr):
    for i in range(len(arr)-1):
        if arr[i] > arr[i+1]:
            return False
    return True


print(isArraySorted([1, 2, 3, 4, 5]))
print(isArraySorted([1, 2, 1, 3, 5, 4]))
